### ISSUE TITLE
Migration to @faker-js/faker 6.0.0

### ISSUE THREAD

**Issue by Shinigami92** [json-schema-faker/json-schema-faker#691]
Hello :wave: 
I'm one of the maintainer of https://github.com/faker-js/faker and we expect to release a beta or RC in the next few days/week.
I would like to create a PR that migrates to the new version, so we can see if anything needs to be done before officially releasing the `6.0.0`.

**Comment by pateketrueke** [comment]
Yeah, sure, I think that switching dependencies would be good, and also painless, just make sure you run the tests. Thank you! :beers:
